src/cli/cmd_clean.py

```diff
@@ -0,0 +1,150 @@
+# pylint: disable=duplicate-code
+"""Module for the cleaning command."""
+import logging
+import sys
+from datetime import datetime
+from pathlib import Path
+from typing import Annotated, Optional
+
+import typer
+
+from src.blueprints.datafile import RawDatafile
+from src.cli.common import (
+    LogFileOption,
+    ProfileNameOption,
+    ProfileVersionOption,
+    SourceDataPathArg,
+    StorageBoxOption,
+    get_config,
+)
+from src.config.config import ConfigFromEnv
+from src.inspector.inspector import Inspector
+from src.profiles.profile_register import load_profile
+from src.utils import log_utils
+from src.utils.filesystem.ctime import max_ctime
+from src.utils.timing import Timer
+
+logger = logging.getLogger(__name__)
+
+
+def _check_verified_status(config: ConfigFromEnv, datafiles: list[RawDatafile]) -> None:
+    logger.info("Retrieving status...")
+    # Check ingestion status for each file.
+    timer = Timer(start=True)
+    insepctor = Inspector(config)
```

Comment:
Do we want to generate the Inspector here or pass the Inspector class as a input into the function?